modules/03_in_memory_storage/README.md

```diff
@@ -1,9 +1,471 @@
 # In memory storage
 
-TODO: write intro description
-TODO: write tutorial
-TODO: address secret key needing to remain secret <https://flask.palletsprojects.com/en/0.12.x/quickstart/#sessions>
-TODO: explain sessions
-TODO: explain app_version injection
+## Goal
 
-Potential additional topics to cover: testing and debugging
\ No newline at end of file
+Implement an interactive web app where you can:
+- upload words to the game dictionary;
+- play a game of guessing a word from the dictionary.
+
+## Tutorial
```

Comment:
You changed the style of this tutortial compared to other tutortials quite a lot. Can you also modify the other tutortials in this style, or converge the style of this to fit more in with the others?

Reply:
That's exactly why I wanted it to be reviewed sooner than later :)
And this PR in its description suggests how to change the structure of the tutorial for project 2, if we go for this style:
https://github.com/MS-SJF-Projects/web-game-complete/pull/8 

Would you agree on this structure - goal/steps as headers/recap + explaining what to do and how, but without details? 
 Do you think it works better, or would you remove certain parts?
 
I think we need a unified style, and that everybody should improve their tutorials themselves, because it looks like a major rewriting. It would be weird / not very team-player of me if I came and violated the sense of ownership by rewriting everything because I like it this or that way :)

Reply:
I think your style looks fine, but I don't want to refactor the other tutorials because you introduced a new style 😆 

Also @LauraDamianTNA should weigh in on her opinions on the two styles :)

Reply:
I agree the styles are different, and I also think they are both clear. I created a WI https://github.com/MS-SJF-Projects/web-game-complete/issues/11 to do this, but I would not make it a prio at this moment and I would focus on finishing what we proposed ourselves (adding DB, pictures, blob storage).